### PR TITLE
[Review] Feature/layout improvements 2

### PR DESCRIPTION
**Pull request by dfosco**
- [x] Improve layout for product page
- [x] Improve layout for product not found 
- [x] Hide product filters when search page comes up empty
- [x] ~~Hide product filters when category page comes up empty~~ (API problem, can't fix for now)

### CODE REVIEW

**Comment by dfosco** [comment]
Update: @tamorim told me on Slack that this is an API issue and can only be solved when their “empty category” response has the correct behavior.

---

@tamorim @amoreira Can one of you help me with the last item on this PR? For some reason Pilates Lovers is showing random products when an unexisting category is accessed:

image

I want it to display the same message the Product List does when it cannot find products to fill the search results:

image

**Comment by dfosco** [comment]
@amoreira @gkozlowaski @tamorim Please review

File: src/components/ProductPage/ProductNotFound/ProductNotFound.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './ProductNotFound.less';
+import SVGIcon from 'utils/SVGIcon/SVGIcon.js';
+import productNotFoundIcon from 'assets/icons/productNotFound_icon.svg';
+import productNotFoundImg from 'assets/icons/productNotFound_icon.png';
+
+class ProductNotFound extends React.Component {
+
```
Comment (tamorim) [inline src/components/ProductPage/ProductNotFound/ProductNotFound.js:8]: @dfosco Unnecessary new line

File: src/components/ProductPage/ProductPage.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import './ProductPage.less';
 import Product from 'components/Product/Product';
+import ProductNotFound from './ProductNotFound/ProductNotFound.js';
```
Comment (tamorim) [inline src/components/ProductPage/ProductPage.js:4]: @dfosco There's no need for the `.js` at the end of the import

File: src/utils/base/custom-theme.less [hunk 3]
```diff
@@ -66,8 +66,9 @@
 .Product {
```
Comment (tamorim) [inline src/utils/base/custom-theme.less:67]: @dfosco Unnecessary new line

File: src/components/Header/HeaderCustom.less [hunk 4]
```diff
@@ -12,7 +12,26 @@
     width: 100%;
   }
 
-  &__menu-inner {
-    .theme-width();
+  &__nav-desktop-inner {
+
```
Comment (tamorim) [inline src/components/Header/HeaderCustom.less:16]: @dfosco Unnecessary new line

File: src/components/SearchHeader/SearchHeader.js [hunk 5]
```diff
@@ -54,14 +34,17 @@ class SearchHeader extends React.Component {
   }
 
   render() {
-    const qty = this.props.qty;
-    const resultMsg = qty === 1 ?
-      `${qty} resultado para` : `${qty} produtos encontrados`;
+
```
Comment (tamorim) [inline src/components/SearchHeader/SearchHeader.js:37]: @dfosco Unnecessary new line

File: src/components/SearchPage/SearchPage.js [hunk 6]
```diff
@@ -23,15 +47,19 @@ class SearchPage extends React.Component {
...
         />
         <div className="SearchPage__content container-fluid">
           <div className="row">
+          {
+            zeroProducts ? null :
             <div className="SearchPage__sidebar hidden-xs hidden-sm col-md-3 col-lg-2">
-              <div className="SearchPage__filter-list">
-                <FilterListSidebar id={this.props.id} location={this.props.location} />
-              </div>
+                <div className="SearchPage__filter-list">
```
Comment (tamorim) [inline src/components/SearchPage/SearchPage.js:57]: @dfosco Wrong indentation on this div

**Comment by tamorim** [comment]
@dfosco Nice work on the product not found page 👍 
After you correct those little things I've noted, I will work to improve some stuff as well

**Pull request merged**